src/VbsEnclaveSDK/src/veil_enclave_lib/userboundkey.vtl1.cpp

```diff
@@ -84,22 +86,22 @@ veil_abi::Types::attestationReportAndSessionInfo userboundkey_get_attestation_re
 {
 
     // DEBUG: Log that the enclave function has been called
-    veil::vtl1::vtl0_functions::debug_print(L"DEBUG: userboundkey_get_attestation_report called - enclave function started");
+    debug_print(L"DEBUG: userboundkey_get_attestation_report called - enclave function started");
```

Comment:
Can we just delete these now? (ideally we'd convert to some tailored tracelogging)